src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -1,47 +1,161 @@
 package core.basesyntax;
 
 import java.util.List;
+import java.util.Objects;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+
+    private Node<T> head;
+    private Node<T> tail;
+    private int size;
+
     @Override
     public void add(T value) {
+        Node<T> newNode = new Node<>(tail, value, null);
```

Comment:
When creating a new `Node<T>`, all the fields are passed to the `Node` constructor. This is good practice.